web/src/App.tsx

```diff
@@ -83,24 +83,20 @@ const App = () => {
 
   useEffect(() => {
     document.documentElement.setAttribute("lang", locale);
```

Comment:
Maybe move this line after `i18n.changeLanguage(currentLocale);`

Reply:
Done :)